src/parser/classic/modules/items/FlaskChecker.ts

```diff
@@ -86,21 +86,21 @@ class FlaskChecker extends Analyzer {
         'You did not have a flask up before combat. Having a flask during combat is an easy way to improve performance.',
       )
         .icon('inv_potion_97')
-        .staticImportance(SUGGESTION_IMPORTANCE.MINOR),
+        .staticImportance(SUGGESTION_IMPORTANCE.MAJOR),
```

Comment:
Not using a any Potions/Flasks is at least an average. Tt's a free way to improve your performance. If you are on WowA to get better, this is the easiest possible way to do so.